drivers/gpu/drm/rcar-rvgc/rcar_rvgc_drv.c

```diff
@@ -284,7 +284,27 @@ static struct rpmsg_driver taurus_rvgc_client = {
 	.callback	= rcar_rvgc_cb,
 	.remove		= rcar_rvgc_remove,
 };
-module_rpmsg_driver(taurus_rvgc_client);
+
+static int __init rcar_rvgc_init(void)
```

Comment:
Well, this is a mere workaround. It may backfire in different system configuration.

Correct solution is to ensure that CR firmware will not send events until new Android instance is connected.

Another correct fix is to fix `rcar_rvgc` driver: why it registers callback when it is not ready to receive it?